app/src/main/java/com/github/h3lp3rs/h3lp/storage/Storages.kt

```diff
@@ -32,9 +36,25 @@ enum class Storages(enableOnlineSync: Boolean) {
          * Reset local storage completely
          */
         fun resetStorage() {
-            for(storage in values()) {
+            for (storage in values()) {
                 storage.ls.clearAll()
             }
         }
     }
+    /**
+     * set the Online synchronization for a given storage
+     * @param isSyncEnable if the synchronization must be enabled
+     */
+    fun setOnlineSync(isSyncEnable : Boolean){
+        getGlobalCtx().getSharedPreferences("SyncPref", AppCompatActivity.MODE_PRIVATE)
```

Comment:
I know this may sound silly, but you can use a storage instance for this ;)

Reply:
yeah I hesitated but it seems bit like a snake biting its own tail